test/groovy/PiperStageWrapperTest.groovy

```diff
@@ -113,7 +113,7 @@ class PiperStageWrapperTest extends BasePiperTest {
         assertThat(executed, is(true))
         assertThat(loggingRule.log, containsString('[piperStageWrapper] Running project interceptor \'.pipeline/extensions/test.groovy\' for test.'))
         assertThat(loggingRule.log, containsString('Stage Name: test'))
-        assertThat(loggingRule.log, containsString('Config: [productiveBranch:master,'))
```

Comment:
more a general question: what is the story behind property `productiveBranch`. Of course I understand we often have a branch for integrating changes/features/fixes. Normally this branch is called `master`. But why is that needed here. E.g. in case we are on a features branch I would expect the feature branchs `HEAD` (or maybe a merge commit between that feature branch and the `master` branch) is checked out and it does not matter that there is another branch ...